packages/esm-appointments-app/src/dashboard.meta.ts

```diff
@@ -1,22 +1,21 @@
-import { Calendar } from '@carbon/react/icons';
-
 export const dashboardMeta = {
-  name: 'appointments',
+  path: 'appointments',
   slot: 'clinical-appointments-dashboard-slot',
   title: 'Appointments',
-};
+  basePath: `${window.spaBase}/home`,
+} as const;
```

Comment:
Using `as const` here instead of explicit typing because no single framework type
has both `basePath` (required by `DashboardExtension`) and `slot` (needed for
extension registration). The const assertion also preserves literal types which
is critical for the strict icon type checking introduced in framework v8.0.1-pre.3349.